kinogoteborg/src/app/components/movie/ReviewForm.jsx

```diff
@@ -12,19 +46,45 @@ export default function Reviews({ isVisible, onClose }) {
         </button>
         <div className=" bg-stone-800 p-2 rounded-md text-stone-200">
           <h2 className="text-2xl text-center">Write a review of your own</h2>
-          <form action="submit" className="flex flex-col gap-2">
+          <form className="flex flex-col gap-2" onSubmit={submitReview}>
             <label htmlFor="Author">Author</label>
```

Comment:
I am not sure if it is supposed to be `<label htmlFor=...>` or if the last "m" is missed in `<label htmlForm=...>`.